lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -1267,34 +1267,46 @@ Float32List toSkColorStops(List<double>? colorStops) {
   return skColorStops;
 }
 
-@JS('Float32Array')
-external _NativeFloat32ArrayType get _nativeFloat32ArrayType;
-
 @JS()
 @staticInterop
-class _NativeFloat32ArrayType {}
+abstract class _NativeArrayType {}
+
+@JS('Float32Array')
+external _NativeArrayType get _nativeFloat32ArrayType;
+
+@JS('Uint32Array')
+external _NativeArrayType get _nativeUint32ArrayType;
 
 @JS('window.flutterCanvasKit.Malloc')
-external SkFloat32List _mallocFloat32List(
-  _NativeFloat32ArrayType float32ListType,
-  int size,
-);
+external Object _mallocList(_NativeArrayType nativeArrayType, int size);
 
 /// Allocates a [Float32List] backed by WASM memory, managed by
 /// a [SkFloat32List].
 ///
-/// To free the allocated array use [freeFloat32List].
+/// To free the allocated array use [freeList].
 SkFloat32List mallocFloat32List(int size) {
```

Comment:
Let's document if `size` is in bytes or element count. Also, if it's the latter, then perhaps `length` is a more idiomatic name for the parameter (and I think `sizeInBytes` is more common for the other one).

Reply:
I'll check what CanvasKit calls the parameter.